bugbug/model.py

```diff
@@ -91,19 +91,29 @@ def classification_report_imbalanced_values(
     return result
 
 
-def get_labeled_confusion_matrix(y_test, y_pred, labels):
-    confusion_matrix = metrics.confusion_matrix(y_test, y_pred, labels=labels)
-    confusion_matrix_table = confusion_matrix.tolist()
-    confusion_matrix_header = []
-    for i in range(len(confusion_matrix_table)):
-        confusion_matrix_header.append(f"{labels[i]} (Predicted)")
-    for i in range(len(confusion_matrix_table)):
-        confusion_matrix_table[i].insert(0, f"{labels[i]} (Actual)")
-    labeled_confusion_matrix = tabulate(
-        confusion_matrix_table, headers=confusion_matrix_header, tablefmt="fancy_grid"
-    )
-
-    return labeled_confusion_matrix
+def print_labeled_confusion_matrix(y_test, y_pred, labels, is_multilabel=False):
+    if is_multilabel:
+        class_names = labels
+        confusion_matrix = metrics.multilabel_confusion_matrix(y_test, y_pred)
+        confusion_matrix_table = confusion_matrix.tolist()
+    else:
+        confusion_matrix = metrics.confusion_matrix(y_test, y_pred, labels=labels)
+        confusion_matrix_table = [confusion_matrix.tolist()]
+
+    for num, table in enumerate(confusion_matrix_table):
+        if is_multilabel:
+            print(f"label: {class_names[num]}")
+            labels = [1, 0]
```

Comment:
Do something like:
```
if is_multilabel:
    print(f"label: {labels[num}")
    table_labels = [1, 0]
else:
    table_labels = labels
```
And then use table_labels later. This way you don't have to define `class_names` above